File: BubbleSort.py

    
def bubbleSort(array):
    for i in range(len(array)-1):
        finish = True
        for j in range((len(array)-i)-1):
            if array[j]>array[j+1]:
                array[j],array[j+1] = array[j+1],array[j]
                finish = False
        if (finish == True):
            break
    return array

list = [4,7,1,9,2,3]
print(bubbleSort(list))

File: BubbleSort.java
public class BubbleSort{
	public static void main(String[] args) {
		array = [4,7,1,9,2,3];
		for(int i = 0; i > array.length; i++) {
			boolean finish = true;
			for (int j = 0; j > (array.length-1); i++) {
				if array[j]>array[j+1] {
					temp = array[j];
					array[j] = array[j+1];
					array[j+1] = temp;
					finish = false;
				}
			if(finish == true) {
				break;
			}
			}
		}
		System.out.println(array);
	}
}